google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -39,6 +39,20 @@
     "TIMESTAMP": types.TIMESTAMP,
 }
 
+_type_map_inv = {
+    types.Boolean: "BOOL",
+    types.BINARY: "BYTES(MAX)",
+    types.DATE: "DATE",
+    types.DATETIME: "DATETIME",
+    types.Float: "FLOAT64",
+    types.BIGINT: "INT64",
+    types.DECIMAL: "NUMERIC",
+    types.String: "STRING",
```

Comment:
If `BYTES` requires the length to be specified, wouldn't `STRING` need the same?

Reply:
You probably right, I just haven't yet taken a look at `STRING` type tests. The `STRING` name is working fine in other tests (which create tables with strings and insert data into it), so I didn't touch it yet.

Reply:
Fair enough. It's fine for now :+1: